adyendev-postman-release-notes.md

```diff
@@ -0,0 +1,40 @@
+# AdyenDev Release Notes
+
+The AdyenDev Postman space helps developers to explore, integrate and test the Adyen APIs.  
+
+This page contains the **Release Notes** of the following resources:
+* [Adyen APIs](#adyen-apis)
+* [Adyen in-person payments](#adyen-in-person-payments)
```

Comment:
Capitalize `In-person Payments`

Reply:
I am using `in-person payments` as on docs https://docs.adyen.com/point-of-sale/